lib/plugins/block_actions.js

```diff
@@ -1,57 +1,57 @@
 const { Vec3 } = require('vec3')
 
-module.exports = inject
-
-const CARDINALS = {
+const CARDINAL_OFFSETS = {
   north: new Vec3(0, 0, -1),
   south: new Vec3(0, 0, 1),
   west: new Vec3(-1, 0, 0),
   east: new Vec3(1, 0, 0)
 }
 
-const FACING_MAP = {
+const CHEST_FACING_MAP = {
   north: { west: 'right', east: 'left' },
   south: { west: 'left', east: 'right' },
   west: { north: 'left', south: 'right' },
   east: { north: 'right', south: 'left' }
 }
 
+module.exports = inject
+
 function inject (bot) {
   const { instruments, blocks } = bot.registry
 
   // Stores how many players have currently open a container at a certain position
   const openCountByPos = {}
 
-  function parseChestMetadata (chestBlock) {
+  bot.parseChestMetadata = (chestBlock) => {
     const chestTypes = ['single', 'right', 'left']
 
     return bot.supportFeature('doesntHaveChestType')
-      ? { facing: Object.keys(CARDINALS)[chestBlock.metadata - 2] }
+      ? { facing: Object.keys(CARDINAL_OFFSETS)[chestBlock.metadata - 2] }
       : {
           waterlogged: !(chestBlock.metadata & 1),
           type: chestTypes[(chestBlock.metadata >> 1) % 3],
-          facing: Object.keys(CARDINALS)[Math.floor(chestBlock.metadata / 6)]
+          facing: Object.keys(CARDINAL_OFFSETS)[Math.floor(chestBlock.metadata / 6)]
         }
   }
 
-  function getChestType (chestBlock) { // Returns 'single', 'right' or 'left'
+  bot.getChestType = (chestBlock) => { // Returns 'single', 'right' or 'left'
```

Comment:
Why is this exposed? If it's for public use it should be in the doc/types. Otherwise you can remove it from bot or make it an undocumented underscore function

Reply:
https://github.com/PrismarineJS/mineflayer/pull/2929#issuecomment-1427924764

I think it's not for public use, but I wanted to ask first

Reply:
Yeah you can just make them underscore functions. If someone wants it or finds a use they can open a PR to document it. Otherwise if it's documented we can't make breaking changes here in the future.